Fix includedir in codec2.pc.in

The template file codec2.pc.in contains the wrong path to the header
files: they are installed in /usr/include/codec2 and not /usr/include/
directly.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/codec2.pc.in
===================================================================
--- a/codec2.pc.in
+++ b/codec2.pc.in
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 libdir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@
-includedir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@/
+includedir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@/codec2
 
 Name: codec2
 Description: A speech codec for 2400 bit/s and below
